docs/conf.py

```diff
@@ -398,3 +398,27 @@ def setup(app):
     f.write(zenodo_record)
 
 print(zenodo_record)
+
+
+# -- Creating tutorials.rst ---------------------------------------------------
+
+notebook_list = []
+
+walk = os.walk("io/")
+
+for root, dirs, fnames in walk:
```

Comment:
```suggestion
for root, dirs, fnames in os.walk("io/"):
```
No need to define it as variable unless you're reusing it